e2e/support/index.ts

```diff
@@ -0,0 +1,15 @@
+declare global {
```

Comment:
Could you add these types in the same file we register these functions? This ensure we don't have to have a single file that contains every custom command type, because over time, it'll become unmanageable.

You can see the example here
https://github.com/metabase/metabase/blob/9bca2f2f646320bb775f54bac50935d898c1bfd3/e2e/support/commands/ui/button.ts#L12-L15

Reply:
Yes, good catch!

Reply:
e0e00b05b15866ba0e29ef0f93043f53959aafa2